packages/core/admin/admin/src/translations/ar.json

```diff
@@ -205,5 +205,688 @@
+    "Settings.apiTokens.ListView.headers.type": "نوع الرمز",
+    "Settings.apiTokens.regenerate": "تجديد",
+    "Settings.apiTokens.title": "رموز API",
+    "Settings.application.customization": "التخصيص",
+    "Settings.application.customization.auth-logo.carousel-hint": "استبدل الشعار في صفحات المصادقة",
+    "Settings.application.customization.carousel-hint": "تغيير شعار لوحة الإدارة (الحد الأقصى للبعد: {البعد} × {البعد} ، الحد الأقصى لحجم الملف: {الحجم} كيلوبايت)",
+    "Settings.application.customization.carousel-slide.label": "شريحة الشعار",
+    "Settings.application.customization.carousel.auth-logo.title": "شعار Auth",
+    "Settings.application.customization.carousel.change-action": "تغيير الشعار",
+    "Settings.application.customization.carousel.menu-logo.title": "شعار القائمة",
+    "Settings.application.customization.carousel.reset-action": "إعادة تعيين الشعار",
+    "Settings.application.customization.carousel.title": "شعار",
+    "Settings.application.customization.menu-logo.carousel-hint": "استبدل الشعار في شريط التنقل الرئيسي",
+    "Settings.application.customization.modal.cancel": "إلغاء",
+    "Settings.application.customization.modal.pending": "شعار معلق",
+    "Settings.application.customization.modal.pending.card-badge": "صورة",
+    "Settings.application.customization.modal.pending.choose-another": "اختر شعارًا آخر",
+    "Settings.application.customization.modal.pending.subtitle": "إدارة الشعار المختار قبل تحميله",
+    "Settings.application.customization.modal.pending.title": "الشعار جاهز للتحميل",
+    "Settings.application.customization.modal.pending.upload": "تحميل الشعار",
+    "Settings.application.customization.modal.tab.label": "كيف تريد تحميل الأصول الخاصة بك؟",
+    "Settings.application.customization.modal.upload": "تحميل الشعار",
+    "Settings.application.customization.modal.upload.cta.browse": "تصفح ملفات",
+    "Settings.application.customization.modal.upload.drag-drop": "قم بالسحب والإفلات هنا أو",
+    "Settings.application.customization.modal.upload.error-format": "تم تحميل تنسيق خاطئ (التنسيقات المقبولة فقط: jpeg ، jpg ، png ، svg).",
+    "Settings.application.customization.modal.upload.error-network": "خطأ في الشبكة",
+    "Settings.application.customization.modal.upload.error-size": "الملف الذي تم تحميله كبير جدًا (الحد الأقصى للبعد: {البعد} x {البعد} ، الحد الأقصى لحجم الملف: {الحجم} كيلوبايت)",
+    "Settings.application.customization.modal.upload.file-validation": "أقصى بُعد: {البعد} × {البعد} ، الحد الأقصى للحجم: {الحجم} كيلوبايت",
+    "Settings.application.customization.modal.upload.from-computer": "من الكمبيوتر",
+    "Settings.application.customization.modal.upload.from-url": "من URL",
+    "Settings.application.customization.modal.upload.from-url.input-label": "URL",
+    "Settings.application.customization.modal.upload.next": "التالي",
+    "Settings.application.customization.size-details": "أقصى بُعد: {البعد} x {البعد} ، الحد الأقصى لحجم الملف: {الحجم} كيلوبايت",
```

Comment:
we need to be careful here the things that are between {} are variables they don't need to be translated